numpyro/distributions/util.py

```diff
@@ -206,6 +206,7 @@ def _binomial(key, p, n=1, shape=()):
 
 
 def binomial(key, p, n=1, shape=()):
+    n = device_get(n)
```

Comment:
I think that `static_args` does not like DeviceArray anymore. Anyway this is no-op if n is numpy array so I guess things should be fine.

Reply:
Hmm..that's quite strange. Is this intended?

Reply:
I am not sure about that. We can file an issue to JAX devs if this makes regression in performance.